### PR TITLE
PEP8 Fixes

### PR DESCRIPTION
**Pull request by ragv**
This is an attempt to sync between numpy's and scikit-learn's version of numpydoc.

Currently I have made numpydoc's version PEP8 compliant to avoid trivial merge conflicts between both versions.

@larsmans Could you take a look at this?

### CODE REVIEW

File: numpydoc/docscrape.py [hunk 1]
```diff
@@ -500,8 +509,8 @@ def __init__(self, cls, doc=None, modulename='', func_doc=FunctionDoc,
             raise ValueError("Expected a class or None, but got %r" % cls)
         self._cls = cls
 
-        self.show_inherited_members = config.get('show_inherited_class_members',
-                                                 True)
+        self.show_inherited_members = \
```
Comment (rgommers) [inline numpydoc/docscrape.py]: no backslashes please, that's bad style. if this is copied from scikit-learn, it should be changed there. If you wanted to get this line to <80 chars, a better way would be

    self.show_inherited_members = config.get(
                'show_inherited_class_members', True)

**Comment by rgommers** [comment]
This looks fine except for the one ``\``. Sounds like a good plan to merge back scikit-learn specific changes - they could probably use this repo as a git submodule just like numpy does.

**Comment by ragv** [comment]
Have removed the `\`. Please take a look now!

> they could probably use this repo as a git submodule just like numpy does.

Thanks! I have referenced your suggestion in a comment...

**Comment by rgommers** [comment]
Looks good to me now, thanks. I'll leave it open for a bit in case someone else wants to comment.

**Comment by larsmans** [comment]
Looks fine, but I'm not sure why you're pinging me. I don't have pull rights on this repo.

**Comment by rgommers** [comment]
OK in it goes. Thanks @ragv

**Comment by ragv** [comment]
Thanks for the merge :)

**Comment by ragv** [comment]
@larsmans Sorry I had incorrectly assumed you had! Thanks for reviewing it anyway :)

**Pull request merged**